app/jobs/transfer_client_job.rb

```diff
@@ -0,0 +1,24 @@
+class TransferClientJob < ActiveJob::Base
+  queue_as :lupo_background
+
+  def perform(symbol, options = {})
+    client = Client.where(symbol: symbol).first
```

Comment:
Why are you not sending the client directly, a typical flow in ActiveJob?

Reply:
Please take a look at ActionJob documentation regarding sending an instance of a class.

Reply:
Also, please explain why there is two nested jobs (TransferClientJob and UpdateProviderIdJob).

Reply:
@mfenner TransferClientJob is transversing all the dois of a client using pagination. There would be a good number of clients that would be done the first page. But there would be some others that will need a number of pages. I think that putting that in a job would be better than blocking the API response until the whole dois transverse is completed.